src/components/FileRepo.js

```diff
@@ -37,7 +39,7 @@ const arrangerStyles = css`
   }
 `;
 
-const AggregationsWrapper = props => {
+const AggregationsWrapper = ({ showAdvancedFacetView = () => {}, ...props }) => {
```

Comment:
You can wrap AggregationsWrapper with `injectState` so you don't need to pass a different callback.